climakitae/core/data_interface.py

```diff
@@ -1198,6 +1313,16 @@ def warnoflargefilesize(da):
                     "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n"
                 )
 
+        def warn_of_empty_data(self):
+            if self.approach == "Warming Level":
```

Comment:
I realized this could have similar logic to the time-based approach's `(len(self.scenario_ssp) > 1)`, since this warning is only important to notice if > 1 warming levels is retrieved.

Reply:
good point; added something to handle that use case!